### PR TITLE
Build: add group information for IDE's interface rendering.

### PR DESCRIPTION
**Pull request by nkcsgexi**

### CODE REVIEW

**Comment by nkcsgexi** [comment]
@swift-ci please test

File: utils/group.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "Utils.swift",
+    "SourceLocation.swift",
+    "SourceLength.swift",
+  ],
+  "Parse": [
+    "SyntaxParser.swift",
+    "IncrementalParseTransition.swift",
+  ],
+  "Internal": [
+    "AtomicCounter.swift",
```
Comment (akyrtzi) [inline utils/group.json]: Is it required that all files are listed ? "AtomicCounter.swift" doesn't contain any public API.

Comment (nkcsgexi) [reply utils/group.json]: It is required unfortunately. But in practice no users would jump to this group by clicking on any APIs.

**Comment by nkcsgexi** [comment]
@swift-ci please test

**Comment by nkcsgexi** [comment]
@swift-ci please test

**Pull request merged**